website/docs/community/bindings.md

```diff
@@ -111,6 +111,10 @@ As a C library, bindings can be made to call H3 functions from different program
 - [crazycapivara/h3-r](https://github.com/crazycapivara/h3-r)
 - [obrl-soil/h3jsr](https://github.com/obrl-soil/h3jsr)
 
+## SQL Server
+
+- [tbbuck/H3.net.SqlClr](https://github.com/tbbuck/H3.net.SqlClr)
+
```

Comment:
Looks like this should be added after SQLite, below, to keep things in alphabetical order

Reply:
Wow, I'm quite impressed my alphabetical skills failed me there 🤦‍♀️ Good call, updated!